docs/core/preview3/tools/dotnet-sln.md

```diff
@@ -0,0 +1,79 @@
+---
+title: dotnet-sln command | Microsoft Docs
+description: The dotnet-sln command provides a convenient option to add, remove, and list projects in a solution file.
+keywords: dotnet-run, CLI, CLI command, .NET Core
+author: spboyer
+ms.author: mairaw
+ms.date: 02/06/2017
+ms.topic: article
+ms.prod: .net-core
+ms.technology: dotnet-cli
+ms.devlang: dotnet
+ms.assetid: e5a72d3e-c14b-4b0a-a978-c5e54a0988c6
+---
+# Name
+
+## dotnet-sln (Tooling RC4)
+
+dotnet-sln -- Modifies a .NET Core solution file.
+
+## Synopsis
+
+```bash
+dotnet sln [<SLN_NAME>] add <project> <project>
+dotnet sln [<SLN_NAME>] add **/**
```

Comment:
You know that people are going to try this in CMD and then file that it doesn't work, right? can we at least put in a caveat or something? Note?